### PR TITLE
database setup file

### PR DESCRIPTION
**Pull request by wchipchase**

### CODE REVIEW

File: BookAndCanvas/SQL/setupDB.sql [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+alter table UserMsg
+add constraint FK_UserMsg_User_Seller 
+	foreign key (SellerId)
+	references Users (Id)
+
+create table Message
+(
+	[Id] int primary key identity(1,1),
+	[MessageId] int not null,
+	[MsgDate] DateTime not null,
```
Comment (silvestreluna) [inline BookAndCanvas/SQL/setupDB.sql]: msgDate - I think we should use a default date method.  `DEFAULT GETDATE() `? This way we don't have to manually insert a date every time a record gets created.

File: BookAndCanvas/SQL/setupDB.sql [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+alter table UserMsg
+add constraint FK_UserMsg_Message_Id 
+	foreign key (MessageId)
+	references Message (Id)
+
+create table Invoice
+(
+	[Id] int primary key identity(1,1),
+	[Date] DateTime not null,
```
Comment (silvestreluna) [inline BookAndCanvas/SQL/setupDB.sql]: Invoice- I think we should use a default date method.  `DEFAULT GETDATE() `? This way we don't have to manually insert a date every time a record gets created.

Comment (wchipchase) [reply BookAndCanvas/SQL/setupDB.sql]: I don't know how to do this. Can you give me an example of how this is used?

**Review by silvestreluna — COMMENTED** [review]
Should we use default date method for dates?

File: BookAndCanvas/SQL/setupDB.sql (hunk 1 above)
Comment (wchipchase) [reply BookAndCanvas/SQL/setupDB.sql]: Again, I do not understand how this method is appliued when I create the database, can you please clarify?

**Comment by silvestreluna** [comment]
👍

**Pull request merged**